lib/js/components/Badges/BadgeScore/BadgeScore.vue

```diff
@@ -20,98 +27,141 @@
+		'color-suffix': $color-warning-text-weak,
 	),
 	'inverted': (
-		'border': $color-inverted-border,
-		'color': $color-inverted-text,
-		'color-additional': $color-inverted-text,
+		'background': $color-default-background,
+		'icon': $color-neutral-icon-weak,
+		'color': $color-neutral-text-weak,
+		'color-suffix': $color-neutral-text-weak,
 	),
 	'neutral': (
-		'border': $color-neutral-border-strong,
+		'background': $color-neutral-background,
+		'icon': $color-neutral-icon-weak,
 		'color': $color-neutral-text-weak,
-		'color-additional': $color-neutral-text-weak,
+		'color-suffix': $color-neutral-text-weak,
+	),
+	'neutral-medium': (
+		'background': $color-neutral-background-medium,
+		'icon': $color-neutral-icon-strong,
+		'color': $color-neutral-text,
+		'color-suffix': $color-neutral-text,
 	),
 );
 
-@mixin setBadgeScoreColor($self, $border, $color, $color-additional) {
-	border-color: $border;
+@mixin setBadgeScoreColor($self, $background, $icon, $color, $color-suffix) {
+	background-color: $background;
 	color: $color;
-	#{$self}__additionalText {
-		color: $color-additional;
+	#{$self}__icon {
+		color: $icon;
+	}
+	#{$self}__suffix {
+		color: $color-suffix;
 	}
 }
 
 .badgeScore {
 	$self: &;
 
 	@include textBold();
-	@include headlineL();
+	@include headlineM();
 
 	@each $color-name, $color-map in $badge-score-colors {
 		&.-#{$color-name} {
 			@include setBadgeScoreColor(
 				$self,
-				map-get($color-map, 'border'),
+				map-get($color-map, 'background'),
+				map-get($color-map, 'icon'),
 				map-get($color-map, 'color'),
-				map-get($color-map, 'color-additional')
+				map-get($color-map, 'color-suffix')
 			);
 		}
 	}
 
 	border-radius: 4px;
-	border-width: 2px;
-	border-style: solid;
 	display: inline-block;
+	flex: 1;
 	min-width: $badge-score-width;
-	padding: $space-xxxs $space-xxxxs;
-	text-align: center;
+	padding: $space-xxxs $space-xxs;
+
+	&__wrapper {
+		display: flex;
+		justify-content: center;
+	}
+
+	&__content {
+		display: flex;
+		align-items: baseline;
+	}
+
+	&__icon {
+		margin-right: $space-xxxxxs;
+	}
+
+	&__suffix {
+		@include headlineS();
+	}
 
 	&.-small {
 		@include headlineS();
 
 		min-width: $small-badge-score-width;
+		#{$self}__suffix {
+			@include headlineXS();
+		}
 	}
 
 	&.-xsmall {
-		@include textXS();
-
+		font-size: $font-size-info-m;
+		line-height: $line-height-info-m;
 		min-width: $x-small-badge-score-width;
-		border-width: 1px;
-		padding: $space-xxxxs;
+		padding: $space-xxxs;
+
+		#{$self}__suffix {
+			font-size: $font-size-info-m;
+			line-height: $line-height-info-m;
+		}
 	}
 }
 </style>
 
 <script lang="ts">
 import { BADGE_SCORE_COLORS, BADGE_SCORE_SIZES } from './BadgeScore.consts';
+import WnlIcon, { ICONS, ICON_SIZES } from '../../Icon';
+import { VueConstructor } from 'vue';
 
 export default {
 	name: 'BadgeScore',
+	components: {
+		WnlIcon,
+	},
 	props: {
 		text: {
 			type: String,
 			required: true,
 		},
-		additionalText: {
+		suffix: {
```

Comment:
This change is backwards incompatible. Do we have a PR in platform to address that?

Reply:
We haven't yet, but I think it's good time to change this property. We decided to name it `suffix` it's more precisely. I decided to do that also of small cases of usage - 5 in Platform. I can prepare PR in platform after when we finished this PR.

Reply:
@kvas-damian I created subtask for this.

Reply:
remember, that we need it ready before we release DS